### ISSUE TITLE
wrong h1 and url with a post

### ISSUE THREAD

**Issue by tianheg** [freeCodeCamp/freeCodeCamp#41341]
**Describe your problem and how to reproduce it:**

1. `h1` tag must be title, but [here](https://www.freecodecamp.org/news/how-to-get-things-done-lessons-in-productivity/#part-ii-what-to-do-when-you-can-t-stop-procrastinating) is in post
2. 404 url at `Read more articles`: If you want more articles, `check out other posts on my blog`. [location](https://www.freecodecamp.org/news/how-to-get-things-done-lessons-in-productivity/#thanks-for-reading)

**Add a Link to the page with the problem:**

1. https://www.freecodecamp.org/news/how-to-get-things-done-lessons-in-productivity/#part-ii-what-to-do-when-you-can-t-stop-procrastinating
2. https://www.freecodecamp.org/news/how-to-get-things-done-lessons-in-productivity/#thanks-for-reading

**Recommended fix, suggestions (how would you update it?):**

1. change h1 to h2
2. maybe this https://www.freecodecamp.org/news/author/lifetechpsych/ , I think you should contact the author to get the correct answer

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

![image](https://user-images.githubusercontent.com/52186032/109810634-bd557400-7c64-11eb-8262-e14ee8b16a37.png)

---

![image](https://user-images.githubusercontent.com/52186032/109810674-cc3c2680-7c64-11eb-80f2-c5f76afe15ae.png)

**Comment by raisedadead** [comment]
@scissorsneedfoodtoo please check.

**Comment by scissorsneedfoodtoo** [comment]
Hi @tianheg, thanks for catching all of these issues.

I just went through the article and fixed the headers. I notice that Ghost can get confused if authors paste stuff in from another rich text editor. That might be what happened with the headers here.

I also fixed the link to point to the author's page on News. We'll reach out to them in case they want to change the link to point to a personal blog.
